app/controllers/admin/users_controller.rb

```diff
@@ -13,24 +13,30 @@ def update
     private
 
     def user_params
-      accessible = %i[
+      accessible = accessible_params
```

Comment:
could we rename `accessible` with either `permitted` or `allowed`? The two suggestions are widely used around the code, and `accessible` feels a little bit misleading